pkg/helpers/helpers.go

```diff
@@ -59,6 +59,14 @@ func IsTrueBoolPointer(b *bool) bool {
 	return false
 }
 
+// IsFalseBoolPointer is a simple boolean helper function for boolean pointers
+func IsFalseBoolPointer(b *bool) bool {
```

Comment:
Question: Do you think IsFalse would suffice as there being a bool pointer is already suggestive?

Reply:
Because we already have lots of places using `IsTrueBoolPointer` it makes the most sense to keep the same naming pattern.